catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -84,19 +84,21 @@
     @Reference
     CatalogFramework catalog;
 
+    ValidatePrinter printer = new ValidatePrinter();
```

Comment:
@lessarderic  I’ve been looking at it with @spearskw and it’s more complicated than it looks. ValidatePrinter does not implement an interface. It was originally a set of helper methods we separated to its own class to break up functionality and not have one overcluttered class. I don’t think we’re able to inject a non-interface. I don’t think it’s necessary though because the purpose of this ticket was just to remove and extra print and fix the help message of the validate command before it gets released